pilot/control/payloads/generic.py

```diff
@@ -61,9 +54,9 @@ def get_es_stageout_gap(self):
 
         :return: seconds.
         """
-        gap = self.__args.location.queuedata['zip_time_gap'] if 'zip_time_gap' in self.__args.location.queuedata else None
+        gap = infosys.queuedata.es_stageout_gap
         if not gap:
-            pledgedcpu = self.__args.location.queuedata['pledgedcpu']
+            pledgedcpu = infosys.queuedata.pledgedcpu
```

Comment:
Wen, why do you need this redundant wrapper (wrapped function to `infosys`)? why not to query `infosys.queuedata.es_stageout_gap` directly from related code? moreover 

infosys already cares about the defaults:

https://github.com/PanDAWMS/pilot2/blob/next/pilot/info/queuedata.py#L112